lib/pages/home/home.dart

```diff
@@ -34,13 +36,16 @@ class _MyHomePageState extends State<MyHomePage> {
   /// The initial list of workouts to be loaded fresh
   /// from the DB.
   ///
-  late Future<List<Workout>> workouts;
+  late Future workouts;
```

Comment:
Shouldn't this stay the same datatype?

Reply:
Resolved in 7760232ce4de89247f58d4fdcd8cbdcb6b000158